content/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue.md

```diff
@@ -20,7 +20,7 @@ shortTitle: Link PR to issue
 ---
 
 > [!NOTE]
-> The special keywords in a pull request description are interpreted when the pull request targets the repository's _default_ branch. However, if the PR's base is _any other branch_, then these keywords are ignored, no links are created and merging the PR has no effect on the issues. **If you want to link a pull request to an issue using a keyword, the PR must be on the default branch.**
+> The special keywords in a pull request description, i.e. the text comment accompanying the title written by the PR author, are interpreted when the pull request targets the repository's _default_ branch. However, if the PR's base is _any other branch_, then these keywords are ignored, no links are created and merging the PR has no effect on the issues. **If you want to link a pull request to an issue using a keyword, the PR must be on the default branch.**
```

Comment:
I can see why you chose to add the definition here, but think that it would be better placed in the text below. People often overlook information in notes, and the purpose of the note is to highlight the behavior of PRs that target other branches.

It's clear that this article hasn't been updated recently, as this note should not what readers see first 🙈 

I'm going to move the definition and the note into the first section of the article. I can't do this with suggestions, so will make a commit to your branch so you can see what I'm proposing.

Reply:
https://github.com/github/docs/pull/35804/commits/ba3fc5c8e68d4d78252a39039319298a3137f061

Reply:
💯 agree. I only chose the note as it was the first mention, didn't feel confident to go so far as to move things around in the article.

And no problem at all w the more extensive edits. I raised this PR fully expecting further improvements -- I mainly raised it to move the conversation forward. Should have made that clearer in my PR description.

Very happy for the docs to wind up in an even better place!!